contracts/Challenge/PLCRVotingChallenge/PLCRVotingChallenge.sol

```diff
@@ -0,0 +1,149 @@
+import '../../Registry/TokenCuratedRegistry.sol';
+import 'openzeppelin-zos/contracts/token/ERC20/ERC20.sol';
+
+/**
+ * PLCRVotingChallenge is a registry challenge that creates a poll on a
+ * PLCRVoting Contract where voters decide whether to keep or accept an
+ * item onto the registry
+ *
+ * This Challenge contract interfaces with PLCRVoting, communicates results to the
+ * registry, and distributes voter rewards
+**/
+contract PLCRVotingChallenge is IChallenge {
+  using SafeMath for uint;
+
+  event ChallengeClosed();
+  event RewardClaimed(uint reward, address voterAddress);
+
+  address public challenger;             // address of the challenger
+  address public listingOwner;           // address of the listingOwner
+  uint public challengerStake;           // amount of tokens challenger staked in challenge
+  TokenCuratedRegistry public registry;  // address of registry
+  PLCRVoting public voting;              // address of PLCRVoting Contract
+  uint public pollID;                    // poll ID for PLCRVoting contract
+  uint public rewardPool;                // pool of tokens to be distributed to winning voters
+  bool public isClosed;                  // signifies whether challenge has been closed
+  uint public voterTokensClaimed;        // total amount of winning tokens voters have received rewards for thus far
+  uint public voterRewardsClaimed;       // total amount of rewards distributed to voters thus far
+
+  mapping(address => bool) public tokenClaims;   // Indicates whether a voter has claimed their reward yet
+
+  // @dev create new challenge
+  // @param _challenger           account that challenged registry item
+  // @param _listingOwner         account of listingOwner
+  // @param _challengerStake      amount challenger staked for challenge
+  // @param _registry             registry contract
+  // @param _plcrVoting           PLCRVoting contract with voting poll
+  // @param _commitStageLength    commitStageLength for PLCRVoting poll
+  // @param _revealStageLength    revealStageLength for PLCRVoting poll
+  // @param _voteQuorum           voteQuorum needed for winning outcome
+  // @param _percentVoterReward   percent of challengerStake rewarded to voters
+  constructor (
+    address _challenger,
+    address _listingOwner,
+    uint _challengerStake,
+    address _registry,
+    address _plcrVoting,
+    uint commitStageLength,
+    uint revealStageLength,
+    uint voteQuorum,
+    uint _percentVoterReward
+  ) public {
+    require(_percentVoterReward <= 100);
+    challenger = _challenger;
+    listingOwner = _listingOwner;
+    challengerStake = _challengerStake;
+    registry = TokenCuratedRegistry(_registry);
+    voting = PLCRVoting(_plcrVoting);
+    pollID = voting.startPoll(voteQuorum, commitStageLength, revealStageLength);
+    rewardPool = (_percentVoterReward.mul(challengerStake)).div(100);
+  }
+
+  // ====================
+  // CHALLENGE INTERFACE:
+  // ====================
```

Comment:
not sure if we should include these block comments. I think including standard comments for each function is sufficient